src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionChecker.java

```diff
@@ -26,4 +29,27 @@ protected LinkedHashSet<Class<? extends BaseTypeChecker>> getImmediateSubchecker
 
     return checkers;
   }
+
+  @Override
+  public void typeProcessingOver() {
+    Map<String, KnownBindingsValue> knownBindings =
+        DependencyInjectionAnnotatedTypeFactory.getKnownBindings();
+    Map<String, Element> injectionPoints =
+        DependencyInjectionAnnotatedTypeFactory.getInjectionPoints();
```

Comment:
Probably in a separate PR, try making these instance fields of the ATF. I think that will work just as well, and having them be static fields makes me a bit uncomfortable; what if a compiler is reused to compile two different projects? (Rare, but plausible!)

Reply:
I think this is a good idea too. I'll create an issue for this.